### PR TITLE
Change root dirs in Win32.

### PR DESCRIPTION
**Pull request by qnighy**
This patch changes root dirs in Win32, because on Windows,
- `%HOME%` is rarely used. `%userprofile%` is set instead.
- Installation directories are movable.

After this patch, SATySFi [sets the root dirs](https://twitter.com/qnighy/status/974919889013022720) to
- `%userprofile%\.satysfi;%SATYSFI_RUNTIME%` in Win32.
- `$HOME/.satysfi:/usr/local/share/satysfi:/usr/share/satysfi` in other
  platforms.

### CODE REVIEW

**Comment by nekketsuuu** [comment]
Are there any reasons to use `SATYSFI_RUNTIME` only in Win32? In #38, nyuichi proposed [an idea](https://github.com/gfngfn/SATySFi/issues/38#issuecomment-366835572) to use variables on all systems.

Also, note that a similar environment variable `SATYSFI_LIB_ROOT` was used in the past. That is deprecated after aa03c8201fac3bb62f11d2f07cb3c53a04e4baab.

**Comment by qnighy** [comment]
> Are there any reasons to use `SATYSFI_RUNTIME` only in Win32? In #38, nyuichi proposed an idea to use a variable on all systems.

Not actually. I was just conservative with other platforms. Let's discuss some now, but I think it's good to defer the final decision until we get some feedbacks.

Possible discussions are:

- Should an environment variable be used on other platforms? I think yes.
- Should the fixed paths (`/usr/[local/]/share/satysfi`) be removed? I think no for now.
- Name of the environment variable? It's a bikeshed. I picked `*_RUNTIME` from vim, but `SATYSFI_LIB_ROOT` also seems reasonable. Which do you like?

**Comment by nekketsuuu** [comment]
> Not actually. I was just conservative with other platforms. Let's discuss some now, but I think it's good to defer the final decision until we get some feedbacks.

Then we can just postpone the discussions since this PR is only for the Windows support. I can create another PR to support the environment variable or something. How about removing `%SATYSFI_RUNTIME%` in this PR if it's unnecessary now?

**Comment by qnighy** [comment]
It's necessary. Please see the link above. I'm not in hurry to merge only part of it, because the whole patch is already applied in qnighy/satysfi-cross-windows.

**Comment by nekketsuuu** [comment]
So [you want](https://twitter.com/qnighy/status/974913342199414784) to change search paths depending on an installed location, right? That makes sense.

Then I agree to [your previous comment](https://github.com/gfngfn/SATySFi/pull/67#issuecomment-373971220). We should use an environment variable on all platforms. Any variable name seems to be OK to me (yes, this is a bikeshed). I personally prefer `SATYSFI_LIB_ROOT`, with no strong reason. Since `SATYSFI_RUNTIME` is also good enough, I think there is no need to change.

To summarize, I agree to:

* Use `SATYSFI_RUNTIME` on Windows and other distros.
* Leave the fixed search path (`/usr/[local/]/share/satysfi`) currently.

Any opinions?

**Comment by qnighy** [comment]
> So you want to change search paths depending on an installed location, right? That makes sense.

Yes, windows binary distributions are usually assumed to be "relocatable" (in the sense of file paths).

I'll wait for opinions and then change `SATYSFI_RUNTIME` to `SATYSFI_LIB_ROOT` if it's more favored.

**Comment by gfngfn** [comment]
I merged your pull request for the time being. If the original name `SATYSFI_LIB_ROOT` is preferable to `SATYSFI_RUNTIME` in the long run (e.g. for releases), we can modify it with ease.

Thank you.
